onyx-database-examples/querying/src/main/java/com/onyxdevtools/example/querying/LazyQueryExample.java

```diff
@@ -0,0 +1,61 @@
+package com.onyxdevtools.example.querying;
+
+import com.onyx.exception.EntityException;
+import com.onyx.exception.InitializationException;
+
+import com.onyx.persistence.factory.PersistenceManagerFactory;
+import com.onyx.persistence.factory.impl.EmbeddedPersistenceManagerFactory;
+import com.onyx.persistence.manager.PersistenceManager;
+import com.onyx.persistence.query.Query;
+import com.onyx.persistence.query.QueryCriteria;
+import com.onyx.persistence.query.QueryCriteriaOperator;
+import com.onyx.persistence.query.QueryOrder;
+
+import com.onyxdevtools.example.querying.entities.Player;
+
+import java.io.File;
+import java.io.IOException;
+
+import java.util.List;
+
+
+/**
+ @author  cosborn
+ */
+public class LazyQueryExample
```

Comment:
Can you add some comments to this class that addresses the benefit of what the lazy query has.  In the event a user just downloads the example project and doesn't go through the tutorial.